### PR TITLE
Added frequency to watcher node

### PR DESCRIPTION
**Pull request by abencz**
This change was sitting around in my Github since the last time I touched this code, should have merged it then.

Rebased and moved the frequency parsing into the try...except in case it disappears when not connected.

### CODE REVIEW

File: wireless_watcher/nodes/watcher_node [hunk 1]
```diff
@@ -37,6 +37,7 @@ class Connection(wireless_msgs.msg.Connection):
           args['link_quality_raw'] = fields['Link Quality']
           num, den = re.split('/', args['link_quality_raw'])
           args['link_quality'] = float(num) / float(den)
+          args['frequency'] = trim(fields['Frequency'], float)
```
Comment (safrimus) [inline wireless_watcher/nodes/watcher_node]: Maybe this line should not be in the try block but instead with the other args which use fields.get() to handle missing data from iwconfig. (The reason for keeping the try block is that 'link_quality' depends on data from 'link_quality_raw'.)

**Comment by mikepurvis** [comment]
I'm not thrilled about an MD5 message change here, but I suppose it's fine as we're the only users of this.

At some point, there should be a proposal made to add wireless survey related messages to sensor_msgs.

LGTM.

File: wireless_msgs/msg/Connection.msg [hunk 2]
```diff
@@ -11,3 +11,4 @@ int16 noise_level  # dBm
 
 string essid
 string bssid
+float64 frequency
```
Comment (mikepurvis) [inline wireless_msgs/msg/Connection.msg]: Is this necessary? What about the frequency as reported in Network? Can that be cross-referenced using essid, or is that just way too much trouble?

Comment (abencz) [inline wireless_msgs/msg/Connection.msg]: I'm not sure. Does scanning continue while we're rolling around? If not, we may not have information on all the networks.

File: wireless_watcher/nodes/watcher_node (hunk 1 above)
Comment (abencz) [inline wireless_watcher/nodes/watcher_node]: I think you may be right.

**Comment by abencz** [comment]
Made changes but realized PR is against the wrong branch. Will reopen and merge.

**Pull request closed without merging**